src/App.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
 import {BrowserRouter as Router, Route, Switch, Redirect} from "react-router-dom";
-import {Login} from "./Login"
+
 import {Dashboard} from "./Dashboard";
-import {Register} from "./Register";
+import Register from "./Register";
+import Login from "./Login"
+import {createAuthProvider} from "react-token-auth";
+
+// export const [useAuth, authFetch, login, logout] =
+// createAuthProvider({
+//     accessTokenKey: 'accessToken',
+// });
+export const [useAuth, authFetch, login, logout] =
+createAuthProvider({
+    accessTokenKey: 'accessToken',
+    refreshTokenKey: 'refreshToken'
+});
 
 
 function App() {
-    const auth = localStorage.getItem('AUTH');
+    const auth = useAuth();
```

Comment:
It is a bit wrong, because auth is always an array with one elements, you can use it in this way:
```
const [logged] = useAuth();
```

Reply:
The problem is that `!auth` at the currentm moment is always `false` and login page will not be shown at all)